src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ISyntaxFacts.cs

```diff
@@ -419,8 +419,6 @@ void GetPartsOfTupleExpression<TArgumentSyntax>(SyntaxNode node,
 
     SyntaxList<SyntaxNode> GetMembersOfTypeDeclaration(SyntaxNode typeDeclaration);
 
-    // Violation.  This is a feature level API.
-    bool ContainsInMemberBody([NotNullWhen(true)] SyntaxNode? node, TextSpan span);
     // Violation.  This is a feature level API.
     TextSpan GetInactiveRegionSpanAroundPosition(SyntaxTree tree, int position, CancellationToken cancellationToken);
```

Comment:
i've marked apis that currently violate the docs of ISyntaxFacts.  i continually try to move things out when i run across them.